src/test/scala/com/github/mlangc/slf4zio/ReadmeExamplesTest.scala

```diff
@@ -22,6 +22,7 @@ object ReadmeExamplesTest extends DefaultRunnableSpec {
         // ...
         for {
           logger <- makeLogger[SomeClass]
+          marker <- makeMarker("[MARKER]")
```

Comment:
What do you think about making a proper `testM`, that just demonstrates working with `Markers`, similar to `testM("Performance Logging - Using the Logging Service")` that demonstrates performance logging?

Reply:
Yes. Good idea to do like you say instead of hooking into each test like i've done. Will fix